### PR TITLE
[cpprestsdk] Update to v2.10.16

### PR DESCRIPTION
**Pull request by BillyONeal**

### CODE REVIEW

File: ports/cpprestsdk/portfile.cmake [hunk 1]
```diff
@@ -3,8 +3,8 @@ include(vcpkg_common_functions)
 vcpkg_from_github(
```
Comment (LilyWangL) [inline ports/cpprestsdk/portfile.cmake:3]: `include(vcpkg_common_functions)` is deprecated. Please remove it.

**Review by LilyWangL — CHANGES_REQUESTED** [review]

**Comment by LilyWangL** [comment]
The way to handle feature can be replaced by [vcpkg_check_features()](https://github.com/microsoft/vcpkg/blob/a3a6530631df905eb5c0e26d0b20d7d548e0c465/docs/maintainers/vcpkg_check_features.md).
```
set(CPPREST_EXCLUDE_BROTLI ON)
if ("brotli" IN_LIST FEATURES)
    set(CPPREST_EXCLUDE_BROTLI OFF)
endif()
set(CPPREST_EXCLUDE_COMPRESSION ON)
if ("compression" IN_LIST FEATURES)
    set(CPPREST_EXCLUDE_COMPRESSION OFF)
endif()
set(CPPREST_EXCLUDE_WEBSOCKETS ON)
if("websockets" IN_LIST FEATURES)
    set(CPPREST_EXCLUDE_WEBSOCKETS OFF)
endif()
```

**Comment by BillyONeal** [comment]
@ras0219-msft Thanks for resolving the CR feedback!

**Review by LilyWangL — APPROVED** [review]

File: ports/cpprestsdk/CONTROL [hunk 2]
```diff
@@ -1,5 +1,5 @@
 Source: cpprestsdk
-Version: 2.10.15-1
+Version: 2.10.16
 Build-Depends: openssl (!uwp&!windows), boost-system (!uwp&!windows),
   boost-date-time (!uwp&!windows), boost-regex (!uwp&!windows), boost-thread (!uwp&!windows),
   boost-filesystem (!uwp&!windows), boost-random (!uwp&!windows), boost-chrono (!uwp&!windows),
```
Comment (cbezault) [inline ports/cpprestsdk/CONTROL:5]: As an aside, I _think_ default features can take qualifiers now.

**Pull request merged**